packages/core/components/FilterDisplayBar/FilterMedallion.tsx

```diff
@@ -5,17 +5,11 @@ import * as React from "react";
 import { useDispatch } from "react-redux";
 
 import { selection } from "../../state";
-import FileFilter from "../../entity/FileFilter";
+import FileFilter, { Filter } from "../../entity/FileFilter";
 import AnnotationFilter from "../AnnotationSidebar/AnnotationFilter";
 
 import styles from "./FilterMedallion.module.css";
 
-export interface Filter {
```

Comment:
Moved this interface out into ../entity/FileFilter since it gets used a couple times